Gateway/Request/CheckoutDataBuilder.php

```diff
@@ -337,7 +335,6 @@ protected function getOpenInvoiceData($order): array
             $formFields['lineItems'][] = [
                 'id' => 'shippingCost',
                 'amountExcludingTax' => $formattedPriceExcludingTax,
-                'taxAmount' => $formattedTaxAmount,
```

Comment:
I thought of removing these items from the payments request because it is not required. The Capture and Refund calls do seem to need it so those are left there.

Reply:
@maassenbas is it similar to what you did in SFCC?

Reply:
In SFCC, we set the taxPercentage=0 instead of removing the taxAmount. 
https://github.com/Adyen/adyen-salesforce-commerce-cloud/pull/456/files

Reply:
Gotcha @maassenbas, I guess you do that for lines that don't have tax? In our case the lines might have tax, we were previously sending the amount and the percentage but I see here that only the percentage is required (at least for capture and refunds) https://docs.adyen.com/payment-methods/klarna/invoice-lines#partial-capture-or-refund

Reply:
@acampos1916 I'm not even sure if it's going to solve the problem or not.
In case you remove the tax amount but keep the line item tax percentage then the tax amount will still be the "same"  because it is calculated by item. Were you able to test it with both scenarios when the tax is calculated on the whole cart and also when it's calculated per item?